oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -150,6 +150,51 @@ ADRs
 .. _invaluable technique: https://www.thoughtworks.com/radar/techniques/lightweight-architecture-decision-records
 .. _edx/credentials/docs/decisions: https://github.com/edx/credentials/tree/master/docs/decisions
 
+A suggested ADR template::
+
+  ##################
+  0000 Title For ADR
+  ##################
```

Comment:
FWIW, this decision is in line with what I'm recommending as a part of the new docs for documentors: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html

Reply:
This generally aligns with the styling in the RST Guides and what python uses with the caveat that they recommend overlines and I don't really mention them because it's doesn't impact the rendered output and is just one more thing to explain to people who are new to RST.

Reply:
I removed the overline to align with @feanil’s doc.

Reply:
Not my first choice, but happier about having clear recommendations. Thanks @feanil.